src/components/item/BulkItemsForm.vue

```diff
@@ -66,6 +78,11 @@ export default {
           itemTitle = itemTitle.slice(0, itemTitle.length - this.trimFiguresFrom.end);
         }
 
+        if (itemTitle.length > this.$options.ITEM_TITLE_MAX_LENGTH) {
+          this.titlesErrorMessage = this.$options.ITEM_TITLE_ERROR_MESSAGE;
+          return;
```

Comment:
line before return
and also return something like false or null that it will be explicit